yogstation/code/modules/mining/lavaland/bloodbook.dm

```diff
@@ -158,6 +158,8 @@
 		if(target.mind)
 			return
 	if(istype(target, /mob/living/simple_animal/hostile/retaliate/goat/king))
+		if(istype(target, /mob/living/simple_animal/hostile/retaliate/goat/king/phase2))
+			return
```

Comment:
but why? istype includes subtypes

Reply:
oops forgot i did this while testing something